safekeeper/src/http/routes.rs

```diff
@@ -320,6 +319,28 @@ async fn timeline_checkpoint_handler(request: Request<Body>) -> Result<Response<
     json_response(StatusCode::OK, ())
 }
 
+/// Make term at least as high as one in request. If one in request is None,
+/// increment current one.
+async fn timeline_term_bump_handler(
```

Comment:
Could you include some "why" aspect in here or in the PR summary?

Reply:
Aah ok, you are still doing rebase style merges and there is rationale on the commit.